metadata-service/restli-client/src/main/java/com/linkedin/entity/client/RestliEntityClient.java

```diff
@@ -307,6 +308,18 @@ public void updateWithSystemMetadata(@Nonnull final Entity entity, @Nullable fin
     sendClientRequest(requestBuilder, authentication);
   }
 
+  public void updateWithSystemMetadata(@Nonnull final Entity entity, @Nullable final SystemMetadata systemMetadata,
+          @Nonnull final Authentication authentication, @Nullable ClientRetry retryPolicy) throws RemoteInvocationException {
+    if (retryPolicy == null) {
+      updateWithSystemMetadata(entity, systemMetadata, authentication);
+    } else {
+      retryPolicy.retry(() -> {
```

Comment:
I see we are adding these nullable retryPolicies - can we just always require a retry policy?